### ISSUE TITLE
New name of the repository

### ISSUE THREAD

**Issue by dmiskiew** [meteortemplates/forms#71]
Hi Jon,

currently it's very hard to find information specific to your code in google, becasue "meteor", "templates" and "forms" terms are too generic.

I think it's a good idea to rename your repository to something more unique. Maybe just add some prefix/suffix?

Thanks!

**Comment by jonjamz** [comment]
You're right. It's been a hectic summer for me and I haven't been able to work on the package as much as I wanted, but now that's coming to an end in the next few weeks, so I'll address this in the next release (what I've started on the `release/2.0.0` branch).

Thank you.

**Comment by shwaydogg** [comment]
perhaps:    **powerForm**

* Em**power** templates with 
   * the most common form tasks
   * keeping up speed
* Em**power** meteor devs to 
   * keep control of their form html, logic and submissions while keeping things sleek and DRY
   * etc...

Logo with "Power" in the flaming meteor debri and "Form" on the meteor.  --> Power of associated with a shooting star (which is a meteor).  Since you had mentioned msavin esque marketing ;) in the meteor forums.

I wouldn't normally have put this in a git issue, but since it was already opened I went ahead

Also, a brief check online didn't yield any other cs packages named powerform that would be confused just some docusign stuff that would be hard to confuse.  I think powerForm rolls of the tongue as it's similar to powerful.
